data/transactions/logic/eval.go

```diff
@@ -2664,7 +2666,7 @@ func opEcdsaVerify(cx *EvalContext) {
 		return
 	}
 
-	if fs.field != Secp256k1 {
+	if fs.field != Secp256k1 && fs.field != Secp256r1 {
```

Comment:
would need another case here for `LogicVersion` vs `fidoVersion`

Reply:
If I'm not mistaken, the previous conditional in the function enforces the field version using the field spec retrieved from ecdsaCurveSpecByField[].